katello/katello-restore

```diff
@@ -1,24 +1,30 @@
 #!/usr/bin/env ruby
 
+require "socket"
 require 'optparse'
 require 'fileutils'
 require 'date'
 
-STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar']
+STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar*']
 ONLINE_BACKUP_FILES = ['mongo_dump', 'candlepin.dump', 'foreman.dump', 'pg_globals.dump']
 OFFLINE_BACKUP_FILES = ['mongo_data.tar.gz', 'pgsql_data.tar.gz']
+FOREMAN_PROXY_CONTENT_BACKUP_FILES = ['mongo_data.tar.gz']
+FOREMAN_PROXY_CONTENT_ONLINE_BACKUP_FILES = ['mongo_dump']
+FOREMAN_PROXY_CONTENT = "foreman-proxy-content" # keep this variable easy to change for backporting
 
 confirmed = false
+@is_foremnan_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@skip_register = false
 
-optparse = OptionParser.new do |opts|
+@optparse = OptionParser.new do |opts|
```

Comment:
I don't see where you moved optparse into methods, it looks like the only change with regards to optparse is that you changed the type of variable it is? Could you double check this? It should very easily be available to every method, as it's declared outside of any other method.